extension/android/src/main/java/org/pytorch/executorch/LlamaModule.java

```diff
@@ -171,11 +171,12 @@ public long prefillPrompt(String prompt, long startPos, int bos, int eos) {
    * @param prompt The text prompt to LLaVA.
    * @param seqLen The total sequence length, including the prompt tokens and new tokens.
    * @param startPos The starting position in KV cache of the input in the LLM.
+   * @param echo indicate whether to echo the input prompt or not
    * @param llamaCallback callback object to receive results.
    * @return The error code.
    */
   public native int generateFromPos(
-      String prompt, int seqLen, long startPos, LlamaCallback callback);
+      String prompt, int seqLen, long startPos, boolean echo, LlamaCallback callback);
 
```

Comment:
Shouldn't `echo` be the last argument?

Reply:
On the Android side echo is not last argument, but from jni side echo is passed into the runner as the last argument.

Reply:
@larryliu0820 latest commits adds this as last parameter in all layers